### PR TITLE
Fix example server and add use gpu instruction in readme

### PR DESCRIPTION
**Pull request by goriri**

### CODE REVIEW

File: examples/server/server.cpp [hunk 1]
```diff
@@ -21,28 +22,30 @@
...
     std::string hostname = "127.0.0.1";
     std::string public_path = "examples/server/public";
     int32_t port = 1337;
     int32_t read_timeout = 600;
     int32_t write_timeout = 600;
 };
 
-struct bark_params {
-    int32_t n_threads = std::min(1, static_cast<int32_t>(std::thread::hardware_concurrency()));
+// struct bark_params {
```
Comment (PABannier) [inline examples/server/server.cpp:34]: rm the comments if no longer used

File: examples/server/server.cpp [hunk 2]
```diff
@@ -54,51 +57,93 @@ void bark_print_usage(char **argv, const bark_params &params) {
     fprintf(stderr, "\n");
 }
 
-void bark_params_parse(int argc, char **argv, bark_params &params) {
+void bark_params_parse(int argc, char **argv, bark_params &params, server_params &server_params)
+{
     bool model_req = false;
-    for (int i = 1; i < argc; i++) {
+    for (int i = 1; i < argc; i++)
+    {
```
Comment (PABannier) [inline examples/server/server.cpp:64]: We favour the curly braces on the same line as the statement. Please revert these changes.
```suggestion
    for (int i = 1; i < argc; i++) {
```

File: examples/server/server.cpp [hunk 3]
```diff
@@ -112,26 +157,43 @@ int main(int argc, char **argv) {
...
         // aquire bark model mutex lock
         bark_mutex.lock();
 
         json jreq = json::parse(req.body);
         std::string text = jreq.at("text");
 
-        // generate audio
         std::string dest_wav_path = "/tmp/bark_tmp.wav";
-        bark_generate_audio(bctx, text.c_str(), params.n_threads);
+        // generate audio
```
Comment (PABannier) [inline examples/server/server.cpp:174]: rm comments

**Review by PABannier — CHANGES_REQUESTED** [review]
Thank you for the PR @goriri ! I have a few comments regarding formatting and cleaning the codebase.

File: examples/server/server.cpp (hunk 3 above)
Comment (goriri) [reply examples/server/server.cpp:137]: Fixed all these as requested.

**Comment by PABannier** [comment]
@goriri Thanks for the PR! :)

**Pull request merged**